.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: Build OpenSearch
       working-directory: ./OpenSearch
-      run: ./gradlew publishToMavenLocal -Dbuild.snapshot=false
```

Comment:
Would you please also change in [CD](https://github.com/opensearch-project/security/blob/main/.github/workflows/cd.yml#L38)?

Reply:
CD should not be changed. CD only runs against release version. This is `SNAPSHOT` version.

Reply:
In this case, the changes in this PR will be reverted upon release? Otherwise, CD would fail.

Reply:
It is expected CD to fail if it is accidentally runs against `SNAPSHOT`.